techniques/system/common/1.0/update.cf

```diff
@@ -74,30 +51,9 @@ body copy_from copy_digest_without_perms(from)
         portnumber => "${system_common.community_port}";
 }
 
-# Use for non-sensitive policy files
-body copy_from remote_unsecured_without_perms(server, path)
```

Comment:
this is used in techniques/system/distributePolicy/1.0/propagatePromises.st - you should remove it there also